check.sh

```diff
@@ -8,139 +8,193 @@
 
 # Note: at the moment, there is some useless recompilation, which could be improved.
 
-# --help menu
-for arg in $@; do
-    if [ "$arg" == "--help" ]; then
-        echo "Usage: check.sh [--double] [<commands>]"
-        echo ""
-        echo "Each specified command will be run (until one fails)."
-        echo "If no commands are specified, all checks are run (no doc; may take several minutes)."
-        echo ""
-        echo "Commands:"
-        echo "    fmt           format code, fail if bad"
-        echo "    clippy        validate clippy lints"
-        echo "    test          run unit tests (no Godot)"
-        echo "    itest         run integration tests (Godot)"
-        echo "    doc           generate docs for 'godot' crate"
-        echo "    dok           generate docs and open in browser"
-        echo ""
-        echo "Options:"
-        echo "    --double      run check with double-precision"
-        echo ""
-        echo "Examples:"
-        echo "    check.sh fmt clippy"
-        echo "    check.sh"
-        echo "    check.sh --double clippy"
-        exit 0
-    fi
-done
-
-firstArg=1
-toolchain=""
-extraArgs=()
-
-if [[ "$1" == "--double" ]]; then
-    firstArg=2
-    extraArgs+=("--features double-precision")
-fi
+################################################################################
+# Constants
+################################################################################
+
+# Commands to run (in that order) if none are given on the command line.
+DEFAULT_COMMANDS=("fmt" "clippy" "test" "itest")
+
+# Store help text in a variable $HELP_TEXT so we don't need weird indentation later on.
+read -r -d '' HELP_TEXT <<EOF
+Usage: check.sh [OPTION|COMMAND...]
+
+Each specified command will be run (until one fails).
+If no commands are specified, the following commands will be run:
+    ${DEFAULT_COMMANDS[@]}
+
+Commands:
+    fmt           format code, fail if bad
+    clippy        validate clippy lints
+    test          run unit tests (no Godot needed)
+    itest         run integration tests (from within Godot)
+    doc           generate docs for 'godot' crate
+    dok           generate docs and open in browser
+
+Options:
+    -h, --help    print this help text
+    --double      run check with double-precision
+
+Examples:
+    check.sh fmt clippy
+    check.sh
+    check.sh --double clippy
+    RUSTUP_TOOLCHAIN=nightly check.sh
+EOF
+
+# Terminal color codes.
+RED='\033[1;31m'
+GREEN='\033[1;32m'
```

Comment:
Very sneaky change! 
Can you revert this to the previous color? Feel free to rename accordingly.

It was deliberately chosen as cyan, also consistent with gdnative's check script.

Reply:
Weird, but OK :)